### PR TITLE
docs: add api docs

### LINKED ISSUE #26: Add documentation

### PR DESCRIPTION
**Pull request by pyphilia**
This PR contains the documentation to use the apps api for apps developers. 

close #26

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Graasp Apps API Guide
+
+Graasp can embed web applications as a resource. They should be added as an item of type `app` to access the Graasp API and get authenticated with a token.
+
+## Introduction
+
+When an app is added as a resource, it is linked to an item. Therefore, two added apps will corredpond to two different items, which leads them to have different item ids, and different app data.
```
Comment (Julien-Torrent) [inline docs/guide.md:7]: correspond

File: docs/api.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+# API
+
+All following requests need an authentication token receive as describe in the [guide](./guide.md).
```
Comment (Julien-Torrent) [inline docs/api.md:3]: received as described

File: docs/api.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+### GET App Action for multiple items
+
+`TODO`
+
+### POST App Action
+
+`POST <apiHost>/app-items/<item-id>/app-action`
+
+- body: `{ data: { ... }, type, [memberId], [visibility] }`
+- returned value: added app data
```
Comment (Julien-Torrent) [inline docs/api.md:42]: created

File: docs/api.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+### GET App Data for multiple items
+
+`TODO`
+
+### POST App Data
+
+`POST <apiHost>/app-items/<item-id>/app-data`
+
+- body: `{ data: { ... }, type, [memberId], [visibility] }`
+- returned value: added app data
```
Comment (Julien-Torrent) [inline docs/api.md:77]: created

**Review by Julien-Torrent — CHANGES_REQUESTED** [review]

**Review by juancarlosfarah — APPROVED** [review]
👍

**Pull request merged**